sympy/functions/combinatorial/tests/test_comb_numbers.py

```diff
@@ -65,6 +65,13 @@ def test_fibonacci():
     assert fibonacci(n).limit(n, S.Infinity) == S.Infinity
     assert lucas(n).limit(n, S.Infinity) == S.Infinity
 
+    assert fibonacci(n).rewrite(sqrt) == \
+        2**(-n)*sqrt(5)*((1 + sqrt(5))**n - (-sqrt(5) + 1)**n) / 5
+    assert fibonacci(n).rewrite(sqrt).subs(n, 10).simplify() == fibonacci(10)
```

Comment:
To save time in the tests one should try to find the shortest method of simplification. simplify() is the most general method that involves a whole series of various trial simplifications many of which do apply. In this case it could be possible to use expand() which goes straight to the point.
For the same reason (to save time) the argument 10 could be replaced with a smaller one (if there is noticeable difference in execution time).